third-party/chpl-venv/README.md

```diff
@@ -21,15 +22,16 @@ Python Packages
 ---------------
 
 The primary packages required by `chpldoc` are: `Sphinx`,
-`sphinxcontrib-chapeldomain`, `sphinx_rtd_theme`, and `virtualenv` (for
-installation). The dependencies for these packages are listed in the next
-section.
+`sphinxcontrib-chapeldomain`, `sphinx-rtd-theme`, and `breathe`. The
+dependencies for these packages are listed in the next section.
 
-The primary packages required by `start_test` are: `argparse`, `PyYAML`, and
-`virtualenv` (for installation).
+The primary packages required by `start_test` are: `PyYAML`, `argcomplete`,
+`filelock`, `setuptools`, and `junitparser`.
 
-The primary packages required by `c2chapel` are: `argparse`, `pycparser`, and
-`virtualenv` (for installation).
+The primary packages required by `c2chapel` are: `pycparser` and `pycparserext`.
+
+The primary packages required by the Chapel Language Server (CLS) are: `pytest`
+and `pytest-lsp`.
```

Comment:
There are other packages required, see chapel-py/pyproject.toml. And its required for CLS, chplcheck, and chapel-py. `pytest` and `pytest-cls` are only required when testing, not by users